taskcluster/tc-all-utils.sh

```diff
@@ -122,25 +122,3 @@ verify_bazel_rebuild()
     exit 1
   fi;
 }
-
-# Should be called from context where Python virtualenv is set
-verify_ctcdecoder_url()
```

Comment:
Maybe worth adding unit test coverage of the similar feature we have in `setup.py` ? Your call.

Reply:
I'm not sure how to test `setup.py`. I guess installing the package and then trying to import the decoder dependency to see if it works?

Reply:
It's still not going to test the entire URL computation part that users will use, though.